test/powershell/engine/Formatting/OutputRendering.Tests.ps1

```diff
@@ -48,7 +48,7 @@ Describe 'OutputRendering tests' {
     }
 
     It 'OutputRendering works for "<outputRendering>" to the pipeline' -TestCases @(
-        @{ outputRendering = 'automatic'; ansi = $true }
+        @{ outputRendering = 'automatic'; ansi = $false }
```

Comment:
Should we have explicit test for automatic behavior?

Reply:
This test is explicit for automatic behavior.  There's a separate test above it for host output and this one for pipeline.